include/ProgramVariable.h

```diff
@@ -42,11 +42,9 @@ class ProgramVariable {
 
   void addAlias(ProgramVariable pv);
 
+  // returns aliases as their string representation
```

Comment:
I still don't feel like this comment explains the difference between these two seemingly-similar methods well enough for me to figure out exactly what I'm getting back. In both cases, you're returning `std::set<std::string>`, so isn't `getAllAliases` returning some string representation, too?

Reply:
So `getNamedAliases` returns the aliases that are named (i.e., are "actual" variables in the C code). Intermediate variables present in the IR like %7, %8 (that may be aliased to a named variable) are unnamed; variables present in the C code like %str are named. 
`getAllAliases` returns both named and unnamed aliases.

Reply:
That's a great explanation. Can you add it as a comment here? Then I think we are good to go on this PR.